### ISSUE TITLE
Is there any possible to implement the Native Ads .  

### ISSUE THREAD

**Issue by Lkirankota** [lreiner/cordova-plugin-startapp-ads#1]
If it works for Native Ads then it will finest one,
its working smooth but , 
some time for 2 -3 hits the app is crash then its working fine.

**Comment by lukasreiner** [comment]
Im already trying to find a solution for native ads.
The problem is that we would need to pass the native Ad Data into the WebView for Ionic and then onAdClick we would need to pass the data back to the cordova plugin.

Im currently working on a solution for this.

**Comment by rsmmukesh** [comment]
Any news about native ads ?

**Comment by lukasreiner** [comment]
Yes as far as I can see you can create Native Ads. But you have to create the component yourself. This is because we can only get the data needed for the native ads. But you have to design the Ad by yourself in the frontend. 

I can create a function here in my plugin to do that, but currently im very busy, but yes it is planned.

**Comment by elkhalifte** [comment]
Still looking for it , Good Plugin 🥰
